templates/index.txt

```diff
@@ -1,12 +1,14 @@
-{{ ('='*15) | term_color('blue') }}
-{{ 'FEDORA' | term_color('white') }} {{ '♥' | term_color('red') }} {{ 'PYTHON' | term_color('white') }}
-{{ ('='*15) | term_color('blue') }}
+{{ ('='*15) | apply(colors.blue) }}
+{{ 'FEDORA' | apply(colors.white) }} {{ '♥' | apply(colors.red) }} {{ 'PYTHON' | apply(colors.white) }}
+{{ ('='*15) | apply(colors.blue) }}
 
 {% for point in points %}
-{{ point.name | term_color('white') }}
+{{ point.name | apply(colors.white) }}
+
 {{ point.content | term_indent }}
 
-    {{ point.link.caption | term_color('blue') }} – {{ point.link.href }}
+    {{ point.link.caption | apply(colors.blue) }} – {{ point.link.href }}
+
 {% endfor %}
 
-{% filter term_color('white') %}See http://fedoralovespython.org{% endfilter %}
+{% filter apply(colors.white) %}See http://fedoralovespython.org{% endfilter %}
```

Comment:
Actually, could one extra newline be added at the end? Apparently, when there is just one `\n` at the end, that one is not part of the response:

```console
$ curl http://127.0.0.1:8003/index.txt
===============
FEDORA ♥ PYTHON
===============


...



See http://fedoralovespython.org$ 
```